package.json

```diff
@@ -59,7 +59,9 @@
     "@google-cloud/nodejs-repo-tools": "^2.0.11",
     "@types/express": "^4.11.0",
     "@types/hapi": "^16.1.11",
+    "@types/is": "0.0.18",
     "@types/koa": "^2.0.43",
+    "@types/lodash.has": "^4.5.3",
```

Comment:
Nit: Seems like this is unused?

Reply:
I looked into it, and its actually used in `src/populate-error-message.ts`.